clients/extension/src/background/handlers/requestHandler.ts

```
@@ -614,15 +616,31 @@ export const handleRequest = (
+          let messageBytes: Uint8Array
+          if (typeof message === 'string' && isHexString(message)) {
+            messageBytes = getBytes(message)
+          } else {
+            messageBytes = new TextEncoder().encode(String(message))
+          }
+          console.log('messageBytes', messageBytes)
+
+          const prefix = `\x19Ethereum Signed Message:\n${messageBytes.length}`
+          const fullMessage = prefix + new TextDecoder().decode(messageBytes)
+          console.log('fullMessage', fullMessage)
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)